### DISCUSSION TITLE
Add SKU field to product entry

### DISCUSSION THREAD

**Discussion by ghost** [jonassiewertsen/statamic-butik#173] [Ideas]
For integrating middleware to back-office accounting software, an SKU field to match online products to those kept in book-keeping inventory would be very beneficial. I have already added it into my own blueprint, but I feel this is a strong enough thing to have built into the product.

**Comment by jonassiewertsen** [comment]
How are devs from other countries feeling about this need?

**Reply by freshface** [reply]
Could  be an optional field
